patch/patches/gn_config.patch

```diff
@@ -64,7 +64,7 @@ index 5121fa721a06..b5ab128722f0 100644
  
      input_locales = locales
 -    output_dir = "${invoker.output_dir}/locales"
-+    output_dir = "${invoker.output_dir}/chrome/locales"
++    output_dir = "${invoker.output_dir}/locales/chrome"
```

Comment:
This change makes it possible to build chrome (otherwise the linker fails to build the binary because there is a chrome subdirectory there) but as we don't build chrome, perhaps this is not needed.